lib/matplotlib/tests/test_quiver.py

```diff
@@ -133,6 +134,20 @@ def test_barbs():
              cmap='viridis')
 
 
+@cleanup
+@raises(ValueError)
```

Comment:
I personally don't care… I much prefer nose's API than pytests, so I have a tendency to use naturally pytest. I'd be interested in knowing whether we should write everything pytest-like.